tests/integration_tests/test_dac/test_dac.py

```diff
@@ -78,11 +78,18 @@ def test_dac_write_and_read_voltages(analog_out, voltage, raises, dac):
         assert voltage_val == pytest.approx(expected_voltage, abs=STORE_ERROR)
         assert code == dac.dac_ops.voltage_to_code(analog_out.value, voltage, dac_gain)
 
-def test_dac_gain(dac):
+@pytest.mark.parametrize(
+    "gain_enable",
+    [(True),
+     (False),
```

Comment:
I don't think you need to put these in tuples since you only have `gain_enable`, but it doesn't affect anything if they are in tuples.